### DISCUSSION TITLE
Custom SSID for receivers

### DISCUSSION THREAD

**Discussion by evilC** [ExpressLRS/ExpressLRS#1669] [Ideas]
With the upcoming Betaflight Configurator via ELRS Wifi feature, would it not make sense to allow us to set a custom SSID for our receivers?
If all you are using it for is to rebind / loan, then it's unlikely you would have two people in close proximity both with WiFi mode activated, but if someone is sat for an extended period configuring their quad via ELRS wifi, it could become confusing when you have multiple APs with the same name. I guess you could probably differentiate via signal strength, but I still feel that if it's fairly trivial to implement this, it would be a nice feature

**Comment by pkendall64** [comment]
It would be better if you had a hotspot on your phone/laptop with the same SSID as your home network so your RX can connect to that instead.
